content/millennium/r4/care-provision/care-team.md

```diff
@@ -35,7 +35,7 @@ Search for CareTeams that meet supplied query parameters:
 
 _Implementation Notes_
 
-* For CareTeam Search operations, both Careteam.read and Encounter.read scopes are required.
+* For CareTeam Search operations, only Careteam.read is required.
```

Comment:
@ramya1096 , can we remove the complete Implementation Notes section, as we had added them to mention Encounter.read scope is required?

cc: @UjwalaBheema , @ashu-bdvt

Reply:
i have removed implementation notes section after confirming with ujwala.